.github/workflows/msys2.yml

```diff
@@ -0,0 +1,39 @@
+name: msys2
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        include: [
+          { msystem: MINGW64, arch: x86_64 },
+          { msystem: MINGW32, arch: i686 }
+        ]
+      fail-fast: false
+    steps:
+      - name: Prepare git
+        run: git config --global core.autocrlf false
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: ${{ matrix.msystem }}
+          install: mingw-w64-${{ matrix.arch }}-toolchain make patch
+          update: true
+      - name: Building TileDB
+        run: cd .github/workflows/mingw-w64-tiledb && makepkg-mingw --noconfirm --syncdeps
+        env:
+          TILEDB_HOME: ${{ github.workspace }}
+          MINGW_INSTALLS: ${{ matrix.msystem }}
+          PKGEXT: ".pkg.tar.xz"
+        shell: msys2 {0}
+      - name: "Upload binaries"
+        uses: actions/upload-artifact@v2
+        with:
+          name: mingw-w64-${{ matrix.arch }}-tiledb
```

Comment:
Could you please add file extension here

Reply:
I don't think that is correct. This works like a bucket name, it can have many files in it.

Reply:
I see, ok.